<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendDialog</class>
 <widget class="QDialog" name="SendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>500</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="to_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Must have the format [local_part]@[domain]</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Email address(es) to send to</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cc_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Address(es) to carbon copy to</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bcc_button">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>80</y>
     <width>93</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bcc</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cc_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cc:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bcc_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Address(es) to blind carbon copy to</string>
   </property>
  </widget>
  <widget class="QLabel" name="bcc_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bcc:</string>
   </property>
  </widget>
  <widget class="QLabel" name="subject_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Subject: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="subject_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Enter subject here</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>701</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SendDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SendDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
